elasticsearch/bulk/bulk.go

```diff
@@ -180,11 +185,47 @@ func (b *Bulk) flushMessages() error {
 func (b *Bulk) bulkRequest() error {
 	startedTime := time.Now()
 	reader := bytes.NewReader(b.batch)
-	_, err := b.esClient.Bulk(reader)
+	resp, err := b.esClient.Bulk(reader)
 	b.metric.BulkRequestProcessLatencyMs = time.Since(startedTime).Milliseconds()
 	if err != nil {
 		return err
 	}
+	err = b.handleResp(resp)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (b *Bulk) handleResp(resp *esapi.Response) error {
```

Comment:
i think method name is too general we can rename it to hasResponseError, checkResponseError

Reply:
I renamed it.